nolearn/lasagne/base.py

```diff
@@ -83,6 +83,17 @@ def __getstate__(self):
         return state
 
 
+def objective(layers,
+              loss_function,
+              target,
+              aggregate=aggregate,
+              **kwargs):
+    output_layer = list(layers.values())[-1]
```

Comment:
So do you still need ` output_layer = list(layers.values())[-1]`? What about making a `get_output_layer`?